src/KubeLibrary/KubeLibrary.py

```diff
@@ -611,6 +611,27 @@ def get_secrets_in_namespace(self, name_pattern, namespace, label_selector=""):
         secrets = [item for item in ret.items if r.match(item.metadata.name)]
         return secrets
 
+    def get_namespaced_pod_exec(self, name, namespace, cmd):
+        """Exec command on selected container for POD.
+
+        Returns command stdout/stderr
+
+        - ``name``:
+          pod name
+        - ``namespace``:
+          namespace to check
+        - ``command``:
+          command to be executed
+        """
+        return stream.stream(self.v1.connect_get_namespaced_pod_exec,
```

Comment:
Would be nice to add container option, if we have more than one container.

Reply:
@m-wcislo 
I have added such an option, which should work, but octopus tests fail. Would be great if you can help to debug it with me ;-)

Reply:
Tests passed, so you made it work right?